task-runner/task_runner/file_manager.py

```diff
@@ -77,18 +77,26 @@ def upload_output(
         task_id: str,
         task_dir_remote: str,
         local_path: str,
+        stream_zip: bool = True,
     ):
         del task_dir_remote  # unused
 
+        if stream_zip:
+            data = files.get_zip_generator(local_path)
+            size = data.total_bytes
+        else:
+            path = os.path.join(local_path, "artifacts")
```

Comment:
When making a zip file I needed to pass the path concatenated with "artifacts", otherwise all the files would be inside the subdirectory "artifacts". 
However, for stream_zip it is not required to pass "artifacts". Does someone know why?

Reply:
It was a problem with the zip function not including some directories, thanks @bgregoinductiva for finding the problem